index.js

```diff
@@ -131,6 +150,13 @@ module.exports = function (session) {
    */
 
   PGStore.prototype.query = function (query, params, fn) {
+    if (this.pgPool !== undefined) {
+      this.pgPool.query(query, params, function (err, res) {
+        if (fn) { fn(err, res && res.rows[0] ? res.rows[0] : false); }
+      });
+      return;
+    }
+
     if (!fn && typeof params === 'function') {
       fn = params;
       params = [];
```

Comment:
This piece of code needs to still be at the top of the `query()` method, if the second parameter is a `function` then it should be treated as the callback – not the parameters – else the syntax of this method is changed.